README.md

```diff
@@ -433,7 +433,9 @@ bad
 #### Transformation Signatures
 | Transform | ADT signature | Function Signature |
 |---|---|---|
+| `arrayToList` | `[ a ] -> List a` | `(a -> [ b ]) -> List b` |
```

Comment:
I was wrong with my initial suggestion. Sorry about that! This should be:

`arrayToList ` 
`[ a ] -> List a`
`(a -> [ b ]) -> a -> List b`

Forgot to take the `a`. My bad.